.github/workflows/prevent-msgid.yml

```diff
@@ -0,0 +1,41 @@
+name: Prevent msgid changes in translations
+
+on:
+  pull_request:
+    paths:
+      - "**/*.po"
+
+jobs:
+  prevent-msgid-changes:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - name: Check for changes between msgid and msgstr
+        run: |
+          git reset origin/main
+          for filename in $(git diff --exit-code --name-only | grep '.po')
+          do
+            echo Checking $filename changes
+            if git diff --exit-code $filename | grep -qE POT-Creation-Date; then
```

Comment:
I think we don't need `-E` for `grep` here since the pattern doesn't use anything special?